gradle.properties

```diff
@@ -4,7 +4,8 @@ org.gradle.jvmargs=-Xmx1G
 # check these on https://fabricmc.net/develop
 minecraft_base_version=1.19
 minecraft_version=1.19.2
-yarn_mappings=1.19.2+build.11
+minecraft_version_full=1.19.2
+yarn_mappings=1.19.2+build.8
```

Comment:
I must also wonder why you downgraded the mappings

Reply:
With litematica added and yarn build 11 gradle reload crashed. At least on my side, so I just used same version as in litematica.

Reply:
I'd ask you to try and make it work with latest yarn, mappings should really not affect the build process in any meaningful way

Reply:
Malilib has issues with yarn build 11 
[crash-2022-09-28_21.36.31-client.txt](https://github.com/gliscowo/isometric-renders/files/9668462/crash-2022-09-28_21.36.31-client.txt)

Reply:
Latest compatible is 10

Reply:
@gliscowo is this good?